### DISCUSSION TITLE
get this kid off the github

### DISCUSSION THREAD

**Discussion by ACrazyTown** [FunkinCrew/Funkin#1808] [General]
like what the fuck is this shit loll 
![image](https://user-images.githubusercontent.com/47027981/134722373-fef957dd-93db-4fa2-afb2-6cb64c0df0de.png)

**Comment by AbnormalPoof** [comment]
> pls draw skid and pump porn

im gonna vomit brb

**Comment by DotWith** [comment]
WEEK 9 IDEA!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Reply by KitOdin** [reply]
what is it?

**Reply by Windows1Door2Daniel** [reply]
> what is it?

![Screenshot_20211104-221222](https://user-images.githubusercontent.com/81389527/140462511-fc553d2c-c5e5-4f80-b1f7-b0553ae8405c.png)

**Comment by Windows1Door2Daniel** [comment]
Hes more useless than me

**Reply by DotWith** [reply]
Windows

**Reply by Windows1Door2Daniel** [reply]
Nah Windows is more useful

But the Door Daniel Edition is useless

**Reply by DotWith** [reply]
> Nah Windows is more useful
> 
> 
> 
> But the Door Daniel Edition is useless

Ports over Door Daniel Edition to Windows 11 Pro Dev Branch

**Comment by Techno274721** [comment]
⨀-⨀
